### ISSUE TITLE
Spacing is wrong when using the Thai language.

### ISSUE THREAD

**Issue by nattyboyz** [YarnSpinnerTool/YarnEditor#110]
The font format is right but the spacing is wrong when using the Thai language.
Thai font contains the upper and lower character that has 0 lengths and it seems like Yarn just adds extra space to each of them.

example word
"สุขีสมบูรณ์"
it will turn into
"สุขีสมบูรณ์    "
(extra space)

![image](https://user-images.githubusercontent.com/686526/70999723-87ce7480-210c-11ea-8f25-fdcdd688d722.png)

**Comment by blurymind** [comment]
Yes unfortunately that is due to Ace editor not supporting non monospaced
fonts. It appears that the default multilangual font we currently have
doesn't support Thai local, so your browser is reverting to another font
which is not monospaced.

I could make it detect the users local and use a monospaced font targeting
Asian languages. Currently the Asian languages are not well covered by the
font we have

**Comment by blurymind** [comment]
A better fix would be to replace Ace with monaco or some other editor that
works better, but that would require a huge refactoring

**Comment by blurymind** [comment]
do you know any monospace font that supports most asian languages and is free?

**Comment by thammawat-s** [comment]
hi.  I came across a free monotype Thai font [here](https://www.thaitux.info/node/574)  TlwgTypott-ttf.tar.gz
